clickfile.py

```diff
@@ -511,53 +511,46 @@ def run(
     cost_reports_dir: str,
 ):
     if not network and name == "ui":
-        network = "devnet"
+        network = EnvName.DEVNET
     if DST_ALLURE_CATEGORIES.parent.exists():
         shutil.rmtree(DST_ALLURE_CATEGORIES.parent, ignore_errors=True)
     DST_ALLURE_CATEGORIES.parent.mkdir()
-    if name == "economy":
-        command = "py.test integration/tests/economy/test_economics.py"
-    elif name == "basic":
-        if network == "mainnet":
-            command = "py.test integration/tests/basic -m mainnet"
-        else:
-            command = "py.test integration/tests/basic"
-        if numprocesses:
-            command = f"{command} --numprocesses {numprocesses} --dist loadgroup"
-        if network == network.DEVNET:
-            command += " --retries 3 --retry-delay 2"
-    elif name == "tracer":
-        command = "py.test -n 5 integration/tests/tracer"
-    elif name == "services":
-        command = "py.test integration/tests/services"
-        if numprocesses:
-            command = f"{command} --numprocesses {numprocesses}"
-    elif name == "compiler_compatibility":
-        command = "py.test integration/tests/compiler_compatibility"
-        if numprocesses:
-            command = f"{command} --numprocesses {numprocesses} --dist loadscope"
-    elif name == "evm":
-        command = "py.test integration/tests/neon_evm"
-        if numprocesses:
-            command = f"{command} --numprocesses {numprocesses}"
-    elif name == "oz":
+
+    commands = {
+        "economy": "py.test integration/tests/economy/test_economics.py",
+        "basic": "py.test integration/tests/basic --dist loadgroup",
+        "tracer": "py.test -n 5 integration/tests/tracer",
+        "services": "py.test integration/tests/services",
+        "compiler_compatibility": "py.test integration/tests/compiler_compatibility --dist loadscope",
+        "evm": "py.test integration/tests/neon_evm",
+        "ui": "pytest ui/tests/website_tests",
+        "oz": "",  # the command is defined in run_openzeppelin_tests()
+    }
+
+    if name in commands:
```

Comment:
I would suggest to change this block from negative perspective:
```suggestion
    if name not in commands:
        raise click.ClickException(f"Test group '{name}' does not exist.")
    command = commands[name]
```

Reply:
fixed